tools/site_and_regional/subset_data

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""
+This is a just top-level skeleton script that calls
+subset_data.py.
+The original code (subset_data.py) is located under
+python/ctsm folder.
```

Comment:
will this still be true after the PR is merged?  If not, maybe remove this note?

Reply:
This is a change coming in with this PR. I think it's good to have this note afterwards. Alternatively, you could have the same note in a README file regarding all of the python modules.

Reply:
Yes. So for all out python code we will have a top level "skeleton" code now with `.py` extension that basically just calls the file containing the real code with `.py` extension. The file containing the code with `.py` extension now resides under `python/ctsm`. This is now standard for all ctsm python codes. Here is the link to the  CTSM python standard. https://github.com/billsacks/ctsm/blob/doc_arguments/doc/design/python_script_user_interface.rst